### PR TITLE
Users/leantk/rpm

### PR DESCRIPTION
**Pull request by leantk**
Adding the ability for the Pom to create an RPM that will install the jar and command to run the install/uninstall for users

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -294,6 +304,96 @@ Licensed under the MIT license. See License.txt in the project root. -->
...
+          <execution>
+            <id>generate-rpm</id>
+            <phase>package</phase>
+            <goals>
+              <goal>rpm</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <release>${version.short}</release>
```
Comment (olivierdagenais) [inline pom.xml]: Would you mind setting the value of the `<version>` element to `${version.short}` and leaving the `<release>` element without a value?  The [documentation](http://www.mojohaus.org/rpm-maven-plugin/ident-params.html#release) implies there's interesting behaviour by default if the `release` parameter isn't set.

File: pom.xml [hunk 2]
```diff
@@ -294,6 +304,96 @@ Licensed under the MIT license. See License.txt in the project root. -->
...
+          <group>Development/Tools</group>
+          <defaultFilemode>755</defaultFilemode>
+          <summary>Git Credential Manager for Mac or Linux</summary>
```
Comment (olivierdagenais) [inline pom.xml]: Are you able to use `${project.name}` for the value of the `<summary>` element?

File: pom.xml (hunk 1 above)
Comment (leantk) [inline pom.xml]: Yes I can remove Release. For some reason I thought it was complaining if it wasn't there but it just worked fine w/o it.

File: pom.xml (hunk 2 above)
Comment (leantk) [inline pom.xml]: Yes good catch!

**Comment by olivierdagenais** [comment]
Thanks for making the changes!

**Pull request merged**